release_notes/unreleased.md

```diff
@@ -1 +1,4 @@
 **Unreleased**
+
+* Compatibility changes for Python 3 support [PAPP-25307]
+* Update Authentication Workflow
```

Comment:
Should we update it to "Updated authentication workflow to utilize refresh token"?